demo/components/victory-tooltip-demo.js

```diff
@@ -172,4 +175,94 @@ class App extends React.Component {
   }
 }
 
+class VictoryTooltipInPortal extends VictoryTooltip {
+  static contextTypes = {
+    portalRegister: React.PropTypes.func,
+    portalDeregister: React.PropTypes.func
+  }
+
+  render() {
+    this.element = super.render();
+    return null;
+  }
+
+  componentDidUpdate() {
+    this.context.portalRegister(this, this.element);
+  }
+
+  componentWillUnmount() {
+    this.context.portalDeregister(this);
+  }
+}
+
+class SvgPortal extends React.Component {
+  componentWillMount() {
+    this.map = new Map();
+    this.portalRegister = this.portalRegister.bind(this);
+    this.portalDeregister = this.portalDeregister.bind(this);
+  }
+
+  portalRegister(key, element) {
+    this.map.set(key, element);
+    this.forceUpdate();
+  }
+
+  portalDeregister(key) {
+    this.map.delete(key);
+  }
+
+  render() {
+    return (
+      <g>
+        {[...this.map.values()].map((el, i) => {
+          return el ? React.cloneElement(el, {key: i}) : el;
+        })}
+      </g>
+    );
+  }
+}
+
+class VictoryContainer extends React.Component {
```

Comment:
@cpb A HOC for this purpose (adding something in the middle of what VictoryContainer renders) would actually be pretty confusing I think, and easy to break.

@boygirl adding this functionality to VictoryContainer behind props is actually what I was thinking for this, but I was thinking of a boolean prop (eg. `withPortal`), why would a portal need a name? there's no reason to have more than one in the same `svg` anyway right?